### DISCUSSION TITLE
AutoDrop

### DISCUSSION THREAD

**Discussion by Copyright47** [artisan-roaster-scope/artisan#1337] [General]
FINALLY upgraded my artisan from a years old verson.   LOVE IT!   I am having an issue I can not figure out.  The program will auto drop at the background's drop TIME.  So say I am running a little behind, it will denote drop, ending the recording of time and development %, making myself and production roasters doing quick math in an important moment of the roast.  'AutoDrop" options has been turned off.  Anyone have a solution for me.  Thank you

**Comment by MAKOMO** [comment]
In the Background dialog (menu Roast >> Background) you might have the "Playback DROP" ticked.
